### DISCUSSION TITLE
Hero 12 requesting updated firmware even though Labs 2.20.70 is installed

### DISCUSSION THREAD

**Discussion by gregb886** [gopro/labs#879] [General]
My new Hero 12 keeps on asking me to update the firmware in the Quik app even though it has 2.20.70 Labs firmware installed successfully.
It's new camera so I it let it install 2.20 official firmware in Quik then restarted then shut down.  Then I reinstalled 2.20.70 Labs firmware but Quik still keeps asking me to update.
What am I missing?

**Comment by dnewman-gpsw** [comment]
Sounds like a big in Quik, anyone else experiencing this?

Have you confirmed that 2.20.70 is shown in the about menu?

**Reply by gregb886** [reply]
Thanks David. I checked this morning and it showed 2.10.70!  It seems I put 2.10.70 Update on the SD Card by mistake.
Now installed 2.20.70 and no problem in Quik.
Totally my fault.  Sorry to trouble you.
